docs/extras/integrations/retrievers/google_cloud_vertex_ai_search.ipynb

```diff
@@ -4,18 +4,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Google Cloud Enterprise Search\n",
+    "# Google Cloud Vertex AI Search\n",
```

Comment:
Yes, it would be less verbose, but I want this to be clear that it's part of Google Cloud.
There's already been lots of confusion from users about the differences between [MakerSuite](https://developers.generativeai.google/products/makersuite) and all of the non-cloud Generative AI Developer Tools.
I can change this if you think it would be an improvement

Reply:
I'll change this to match the other Vertex AI integrations https://python.langchain.com/docs/integrations/llms/google_vertex_ai_palm